en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -70,8 +70,7 @@ The `$searchable_fields` property uses a mixed array format that can be used to
 system. The default is a set of array values listing the fields.
 
 [info]
-`$searchable_fields` will default to use the [`$summary_fields` config](#summary-fields) if not defined. This works fine unless 
-your `$summary_fields` config specifies fields that are not stored in the database.
+`$searchable_fields` will default to use the [`$summary_fields` config](#summary-fields) if not defined. Do not specify fields that are not stored in the database in `$searchable_fields` since it can invoke error.
```

Comment:
I think this is actually more confusing than the old message.

The old message warns that `searchable_fields` will fall back to `summary_fields` - and that if there are non-database-fields in `summary_fields`, that _fallback_ scenario will fail.

The new message warns that `searchable_fields` will fall back to `summary_fields`, but then tells developers not to put non-database-fields in `searchable_fields`.... which they're _not_ doing if they're relying on this fallback behaviour.

It feels like maybe the intention is to keep the old message as it is, but to also add a _separate_ warning against putting non-database-fields in `searchable_fields`?

Reply:
Since the goal of my PR is to exclude from `searchable_fields` those fields that are in `summary_fields` that pass the `hasMethod` check, but do not pass the check for the presence of fields in the DB. Following this logic, now it will not matter whether auxiliary methods like `CMSThumbnail` or `Count` are specified in `summary_fields`, as they will be excluded from `searchable_fields`. Perhaps we could add an appropriate check for `searchable_fields` too to avoid errors. But in any case, the user should be warned.

Reply:
Updated. Try to make it more clear.